Tests/Editor/Tracking/Collision/CollisionIgnorerTest.cs

```diff
@@ -0,0 +1,193 @@
+﻿using Zinnia.Tracking.Collision;
+using Zinnia.Data.Collection;
+
+namespace Test.Zinnia.Tracking.Collision
+{
+    using UnityEngine;
+    using UnityEngine.TestTools;
+    using System.Collections;
+    using NUnit.Framework;
+
+    public class CollisionPointContainerTest
+    {
+        private GameObject containingObject;
+        private CollisionIgnorer subject;
+
+        [SetUp]
+        public void SetUp()
+        {
+            containingObject = new GameObject();
+            containingObject.SetActive(false);
+            subject = containingObject.AddComponent<CollisionIgnorer>();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            Object.DestroyImmediate(containingObject);
+        }
+
+        [UnityTest]
+        public IEnumerator CollisionsIgnored()
+        {
+            WaitForFixedUpdate collisionOccurredYieldInstruction = new WaitForFixedUpdate();
```

Comment:
Given this naming I first thought you created a custom `YieldInstruction` subclass that waits until the collision flag is set!

Perhaps we rename this?
```suggestion
            WaitForFixedUpdate waitForFixedUpdate = new WaitForFixedUpdate();
```

Reply:
the reason it was named the way it was, because the WaitForFixedUpdate instruction kept changing whilst i found a good actual instruction that was consistent. 

But fixed update is consistent so i'm happy to rename the variable